browser/net/BUILD.gn

```diff
@@ -31,18 +31,15 @@ source_set("net") {
 
   deps = [
     "//base",
+    "//brave/app:brave_generated_resources_grit",
     "//brave/browser/safebrowsing",
     "//content/public/browser",
     "//content/public/common",
     "//extensions/common:common_constants",
     "//net",
   ]
 
-  if (!is_android) {
-    deps += [
-      "//brave/app:brave_generated_resources_grit",
-    ]
-  } else {
+  if (is_android) {
```

Comment:
this is the correct fix for the current problem, but if we need `"//brave/app/android:common_extensions",` then we're also missing a non-android dep for `//extensions/common` so I think we should add it now while we're here :)

Reply:
I pushed the fix for it